crates/chia-consensus/src/test_generators.rs

```diff
@@ -255,8 +255,14 @@ fn run_generator(#[case] name: &str) {
     let generator = hex::decode(generator).expect("invalid hex encoded generator");
 
     let expected = match expected.split_once("STRICT:\n") {
-        Some((c, m)) => [c, m],
-        None => [expected, expected],
+        Some((c, m)) => match m.split_once("COSTED_SHA:\n") {
+            Some((d, n)) => [c, d, n],
+            None => [c, m, c],
+        },
+        None => match expected.split_once("COSTED_SHA:\n") {
+            Some((d, n)) => [d, d, n],
+            None => [expected, expected, expected],
+        },
```

Comment:
instead of duplicating all of the output, it might be a good idea to *just* have the alternative costs when this flag is set. Some tests have very large outputs that would be a shame to have to duplicate

Reply:
but if we are not expecting a difference and find one, then that *should* be flagged up, right?

Reply:
This flag is never expected to affect the output, right? It's only affecting cost. So the test cases could specify two costs, one with and one without this flag. That would be a lot more concise than to duplicate all of the output. It would also be easier to manually verify the test cases.